src/views/examples/form-elements/index.njk

```diff
@@ -107,5 +107,91 @@
     <input class="govuk-c-file-upload" type="file" id="file-upload-a">
   </fieldset>
 
+  <div class="govuk-c-radios">
+    <fieldset class="govuk-c-fieldset">
+      <legend class="govuk-c-fieldset__legend">
+        <h1 class="govuk-heading-xl">Have you changed your name?</h1>
+        <span class="govuk-c-fieldset__hint">This includes changing your last name or spelling your name differently.</span>
+      </legend>
+      <div class="govuk-c-radios__item">
+        <input class="govuk-c-radios__input" id="changed-name-1" name="changed-name" type="radio" value="yes" checked>
+        <label class="govuk-c-label govuk-c-radios__label" for="changed-name-1">
+          Yes
+        </label>
+      </div>
+        <div class="govuk-c-radios__conditional">
+          <label class="govuk-c-label" for="previous-first-name">
+          Previous first name
+          </label>
+          <input class="govuk-c-input" id="previous-first-name" name="previous-first-name" type="text">
+
+          <label class="govuk-c-label" for="previous-last-name">
+          Previous last name
+          </label>
+          <input class="govuk-c-input" id="previous-last-name" name="previous-last-name" type="text">
+        </div>
+      <div class="govuk-c-radios__item">
+        <input class="govuk-c-radios__input" id="changed-name-2" name="changed-name" type="radio" value="no">
+        <label class="govuk-c-label govuk-c-radios__label" for="changed-name-2">
+          No
+        </label>
+      </div>
+    </fieldset>
+  </div>
+
+  <div class="govuk-c-radios">
+    <fieldset class="govuk-c-fieldset">
+      <legend class="govuk-c-fieldset__legend">
+        <h1 class="govuk-heading-xl">Legend h1</h1>
```

Comment:
If this is going in the review app, can we use something more descriptive here. @dashouse I know you said the content hasn't been looked at - just flagging it 🙂